MacaoProject/Macao/Models/Game.cs

```diff
@@ -15,10 +15,16 @@ namespace Macao
 {
     public class Game
     {
-        public Player PlayerTurn { get; set; }
+        public Player PlayerTurn { get; set; } = new Player();
+
+        public Card StartCard { get; set; }
 
         public Deck StartDeck { get; set; }
 
+        public Deck Player1Deck { get; set; } = new Deck();
```

Comment:
why player 1 deck is here and not in the class player? i think the deck of the player 1 should be in the class player and maybe here create the two players, because this Game is a two player game right?